lib/ex_doc/formatter/html/templates/js/app.js

```diff
@@ -1,13 +1,350 @@
-;$(function () {
+/*jslint browser: true, es5: true */
+/*globals $, hljs, sidebarNodes: true */
+(function () {
+    "use strict";
+
+    // full_list.js
+    var inSearch = null,
+        defaultSearchItemTimeOut = 0, //set to "0" if not testing
+        searchIndex = 0,
+        searchCache = [],
+        searchString = '',
+        regexSearchString = '',
+        caseSensitiveMatch = false,
+        ignoreKeyCodeMin = 8,
+        ignoreKeyCodeMax = 46,
+        commandKey = 91,
+        lastRowClass = '',
+        sidebarNav = $('.nav'),
+        clicked = null;
+
     // Fix outside world links
     function fixOutsideWorldLinks() {
-      $('a').each(function() {
-        if (window.location.host != this.host) this.target = '_parent';
-      });
+        $('a').each(function () {
+            if (window.location.host !== this.host) {
+                this.target = '_parent';
+            }
+        });
     }
 
     fixOutsideWorldLinks();
 
+    RegExp.escape = function (text) {
```

Comment:
@Dignifiedquire Thanks for your feedback, as you mentioned it was old code, but I already applied your suggestion.